tests/mocks/flask_test_client.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+from app import create_app
+
+
+@pytest.fixture()
+def flask_test_client():
```

Comment:
🤔 Try using pytest-flasks client fixture,

Reply:
Will do in future - show me how